classes/person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Also the same thing go here.

Reply:
@dasileker So I implemented the new code like this:

def add_rentals(date, person)
https://github.com/rentals << Rental.new(date, self)
end

Is this implementation right?